### PR TITLE
patternize validation of axes count with typo fix

### PR DESCRIPTION
**Pull request by PzaThief**
in view functions axis validation and store variable codes  have many errors of below situations and have problem about hard to read.
1. mismatch between validation code and unpacking to store variable
2. mismatch between validation code and console print message
4. mismatch between validation code and parameter docs
etc.

So I made pattern about validation of axes count and fix typos.
I tried to check error of my commit as far as possible, but there can be errors. So please let me know if anyone finds errors.

### CODE REVIEW

**Comment by DidierRLopes** [comment]
I like what you did there! @PzaThief 🚀

**Review by piiq — CHANGES_REQUESTED** [review]
Nice one 🙌
I've left a request for type-hints and a docstring

File: openbb_terminal/helper_funcs.py [hunk 1]
```diff
@@ -1497,6 +1497,32 @@ def __call__(self, parser, namespace, values, option_string=None):
     return ActionClass
 
 
+def is_valid_axes_count(
+    external_axes, n, custom_text=None, prefix_text=None, suffix_text=None
+):
```
[the remark this answers is not in the document]
Comment (PzaThief) [reply openbb_terminal/helper_funcs.py:1506]: > Can you add type-hints and a docstring please?

Oh. I missed it. Thank you for check it. :D

**Comment by Chavithra** [comment]
Fix conflict + CI errors
@piiq I let you check again before merging

**Review by piiq — APPROVED** [review]

**Comment by piiq** [comment]
thanks @PzaThief

**Pull request merged**